src/renderer/components/ft-toggle-switch/ft-toggle-switch.vue

```diff
@@ -11,12 +11,12 @@
     >
     <label
       :for="id"
-      class="switch-label"
+      :class='(compact ? "compact " : "")+"switch-label"'
```

Comment:
Static class names should be separated from dynamic ones.  You can use [this](https://github.com/FreeTubeApp/FreeTube-Vue/blob/2533161667de5bc4ac3657d1f925e6b815c4a416/src/renderer/components/side-nav/side-nav.vue#L4) as an example for syntax.  It is the recommended syntax in the Vue documentation for [Class and Style Bindings](https://vuejs.org/v2/guide/class-and-style.html).